### DISCUSSION TITLE
Bottlerocket debuginfod service

### DISCUSSION THREAD

**Discussion by PowerSurj** [bottlerocket-os/bottlerocket#4270] [General]
Hello,

we have deployed a generic continuous profiler Parca in ebpf mode. Service relies on debuginfod sources to obtain the symbols.
The default debuginfod server used by Parca is: https://debuginfod.elfutils.org/
I don't think it covers bottlerocket binaries. Does bottlerocket provide own debuginfod service?

Thanks!

**Comment by Sparksssj** [comment]
Thank you for your inquiry about Bottlerocket's debuginfod service. At present, Bottlerocket does not provide its own debuginfod service.
While we don't have this capability yet, we'd be happy to create an [issue](https://github.com/bottlerocket-os/bottlerocket-sdk/issues/219) in our project tracker to consider implementing a Bottlerocket-specific debuginfod service in the future.

**Reply by PowerSurj** [reply]
That would be great! Much appreciated!
